mpas_analysis/ocean/time_series_antarctic_melt.py

```diff
@@ -309,13 +316,20 @@ def run_task(self):
 
         regionNames = decode_strings(dsRegionMask.regionNames)
 
+        totalFluxVar = 'timeMonthly_avg_landIceFreshwaterFluxTotal'
+        landIceFluxVar = 'timeMonthly_avg_landIceFreshwaterFlux'
+
+        if totalFluxVar in dsIn:
+            fluxVar = totalFluxVar
+        else:
+            fluxVar = landIceFluxVar
```

Comment:
```suggestion
        fluxVar = self.variableList[0]
```
Seems like there is little harm in this because here it's always of length 1.

Reply:
Done.